### PR TITLE
Added tests to verify error logging

### PR DESCRIPTION
**Pull request by edwardry**
I am aware that my branch is named mock-logger and that I did not actually mock any loggers.

Also. I put two different ways of verifying all the log output. One with parameterized testing and another that just loops through all the possible errors in a single test and asserts against each one. The reason I did this is because I wasn't sure which approach to use. Or perhaps a completely different approach. The upside of parameterized tests is that it explicitly lists each assertion as a separate test and they all run and give you pass/fail. The downside is that it executes the code each time and reads the log file each time. The other method just executes the code once and reads the log file once and validates everything afterward

### CODE REVIEW

File: src/test/java/nl/tistis/pbn/PbnVerifierTest.java [hunk 1]
```diff
@@ -2,76 +2,343 @@
 
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.ValueSource;
```
Comment (folkengine) [inline src/test/java/nl/tistis/pbn/PbnVerifierTest.java]: This line is unused.

**Review by folkengine — CHANGES_REQUESTED** [review]
Dig it!

File: src/test/java/nl/tistis/pbn/PbnVerifierTest.java [hunk 2]
```diff
@@ -2,76 +2,343 @@
...
+import java.util.Map;
+import java.util.stream.Stream;
 
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.mockito.Mockito.*;
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
 
 @DisplayName("PbnVerifier is the main method for the application")
```
Comment (folkengine) [inline src/test/java/nl/tistis/pbn/PbnVerifierTest.java]: Can you change this to just `("PbnVerifier")` ?

**Review by folkengine — CHANGES_REQUESTED** [review]

File: src/test/java/nl/tistis/pbn/PbnVerifierTest.java (hunk 2 above)
Comment (edwardry) [reply src/test/java/nl/tistis/pbn/PbnVerifierTest.java]: ya

**Review by folkengine — APPROVED** [review]

**Pull request merged**